src/avatar/styles.scss

```diff
@@ -0,0 +1,45 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use './mixins.scss' as mixins;
+@use '../internal/styles' as styles;
+@use '../internal/styles/tokens' as awsui;
+@use '../internal/hooks/focus-visible' as focus-visible;
+
+.root {
+  @include styles.styles-reset;
+}
+
+.avatar {
+  display: inline-flex;
+  @include mixins.border-radius-avatar;
+  color: white;
+  block-size: 28px;
```

Comment:
The 28px variable is repeating here and in the dancing dots styles. You can move it to a variable or maybe you can use width,height=100% for dancing dots wrapper.

Reply:
Using width, height 100& caused disruption in the animation but I changed block and inline size to use inherit instead of 28px